app/controllers/memberships_controller.rb

```diff
@@ -12,9 +12,16 @@ def show
 
   # GET /memberships/new
   def new
+    @group = Group.all.find(params.fetch("group_id"))
```

Comment:
`all` is implied and not necessary when writing query methods.

```suggestion
    @group = Group.find(params.fetch("group_id"))
```